go.mod

```diff
@@ -6,7 +6,10 @@ require (
 	github.com/urfave/cli v1.22.5
 )
 
-require github.com/pkg/errors v0.8.1 // indirect
+require (
+	github.com/google/uuid v1.3.0 // indirect
+	github.com/pkg/errors v0.8.1 // indirect
```

Comment:
Oh, I hadn't noticed that errors was in there.  We should remove that.  (That's not part of your PR, just noticed it here)

Reply:
I am not sure how to do that? 
github.com.apex/log depends on errors...

Reply:
Oh, does it?  That's surprising, since errors is deprecated.  But good - so go must have auto-added it rather than (as I'd thought) my having added it and forgotten about it.  Thanks.